Changes.md

```diff
@@ -1,3 +1,16 @@
+1.0.x.x ( relative to 1.0.3.0)
+=======
+
+Improvements
+------------
+
+- TweakPlug : Added the ability to set `InternedStringData` from `StringData` and `StringData` from `InternedStringData`.
```

Comment:
As mentioned elsewhere, I don't think the latter is true.

Reply:
Fixed in 46c6d369866615b65cb896663b6a21591da9fa64